apps/app/src/pages/Repository/Tests.tsx

```diff
@@ -0,0 +1,563 @@
+    </MagicTooltip>
+  );
+};
+
+const DailyVariationGraphCell = ({
+  dailyChanges,
+  totalBuilds,
+}: {
+  dailyChanges: { date: Date; count: number }[];
+  totalBuilds: number;
+}) => {
+  return (
+    <ListCell className={secondaryColumnClassNames[0]}>
+      <div className="items-bottom flex h-8 flex-auto justify-between gap-1">
+        {dailyChanges.map(({ date, count }) => {
+          const height = Math.round((count * 100) / totalBuilds);
+          const heightClassName = `h-${height ? `[${height}%]` : "px"}`;
+          return (
+            <MagicTooltip
+              key={String(date)}
+              tooltip={`${moment(date).format("LL")}: ${count} variation${
+                count > 1 ? "s" : ""
+              }`}
+            >
+              <div className="flex h-full w-full flex-col justify-end">
+                <div className={clsx(heightClassName, "bg-neutral-100")} />
+              </div>
+            </MagicTooltip>
+          );
+        })}
+      </div>
+    </ListCell>
+  );
+};
+
+const VariationsCell = ({
+  dailyChanges,
+  totalBuilds,
+}: {
+  dailyChanges: { date: Date; count: number }[];
+  totalBuilds: number;
+}) => {
+  const totalChanges = dailyChanges.reduce((sum, { count }) => sum + count, 0);
+  return (
+    <ListCell className={secondaryColumnClassNames[1]}>
+      <MagicTooltip
+        tooltip={`Over the last 7 days: ${totalChanges} screenshot change${
+          totalChanges > 1 ? "s" : ""
+        } / ${totalBuilds} total build${totalBuilds > 1 ? "s" : ""}`}
+      >
+        <div>
+          <span className="mr-2 text-lg font-bold">{totalChanges}</span> /
+          <span className="ml-0.5 font-medium text-on-light">
+            {totalBuilds}
+          </span>
+        </div>
+      </MagicTooltip>
+    </ListCell>
+  );
+};
+
+const StabilityCell = ({
+  score,
+  unstable,
+}: {
+  score: number | null;
+  unstable: boolean;
+}) => (
+  <ListCell className={secondaryColumnClassNames[2]}>
+    <MagicTooltip
+      tooltip={"A test with a stability score lower than 60 is unstable"}
+    >
+      <div>
+        <span
+          className={clsx(
+            unstable ? "text-pending-400" : "",
+            "mr-2 text-lg font-bold"
+          )}
+        >
+          {score}
+        </span>
+        /<span className="ml-0.5">100</span>
+      </div>
+    </MagicTooltip>
+  </ListCell>
+);
+
+const TestRow = memo(({ test }: { test: Test }) => {
+  const { selectedTests, toggleTestSelection } = useSelectedTestsState();
+
+  return (
+    <ListRow>
+      <div className="">
+        <input
+          type="checkbox"
+          checked={selectedTests.has(test.id)}
+          onChange={(e) => toggleTestSelection(test.id, e.target.checked)}
+        />
+      </div>
+      <div className="w-95 flex grow gap-4">
+        <Thumbnail screenshot={test.screenshot} />
+        <div className="flex flex-col justify-start gap-1">
+          <div className="flex min-h-[28px] items-start gap-2">
```

Comment:
We don't use px, but also why min-height 28?

Reply:
I made an adjustment to ensure that the date remains in place even when a FlakyChip is displayed. To achieve this, I added a minimum height of 28px. Since Tailwind's utility classes don't include a min-h-7 option, I used pixels instead.

Reply:
Then use rem unit since Tailwind uses rem too.